slides/04-monitor.qmd

```diff
@@ -194,6 +378,46 @@ countdown(minutes = 7)
 ```
 
 
+## ML metrics ➡️ organizational outcomes
+
+::: panel-tabset
+
+### R
+
+```{r}
+augment(v, inspections_new) |> 
+  group_by(facility_type) |> 
+  sensitivity(results, .pred_class)
+```
+
+### Python
+
+```{python}
+## TODO for Isabel: compute sensitivity by facility type for monitoring data
```

Comment:
@isabelizimm can you fill in this chunk, to be like the one above for R?

Reply:
done